distributed-configuration/getting-started-with-etcd/index.md

```diff
@@ -145,6 +145,34 @@ $ curl -L http://127.0.0.1:4001/v2/keys/foo-service?wait=true\&recursive=true
 {"action":"set","node":{"key":"/foo-service/container2","value":"localhost:2222","modifiedIndex":23,"createdIndex":23}}
 ```
 
+### Watching the Directory and triggering an executable
+
+Now let's try watching the `foo-service` directory for changes and - if there are any - run the command. In one window, start watching the directory and in the other window, add another key `container3` with the value `localhost:2222` into the directory. This command shouldn't trigger anything until the key has changed. The same events as in the previous example can trigger a change. The `exec-watch` command expects `etcdctl` to run continuously (for `watch` command you can use `--forever` option)
+
+```sh
+$ etcdctl exec-watch --recursive /foo-service -- sh -c 'echo "\"$ETCD_WATCH_KEY\" key was updated to \"$ETCD_WATCH_VALUE\" value by \"$ETCD_WATCH_ACTION\" action"'
+
+```
+
+In the other window, let's imagine a new container has started and announced itself to the proxy by running:
+
+```sh
+$ etcdctl set /foo-service/container3 localhost:2222
+localhost:2222
+```
+
```

Comment:
I was confused by having both the etcdctl and curl examples. I think we should remove the curl one since we cover that earlier in the doc.